test/benchmark/test-benchmark-async-hooks.js

```diff
@@ -13,6 +13,8 @@ const runBenchmark = require('../common/benchmark');
 runBenchmark('async_hooks',
              [
                'method=trackingDisabled',
-               'n=10'
+               'n=10',
+               'asyncHooks=all',
+               'connections=50'
```

Comment:
Optional micro-nit: We usually keep these in alphabetical order. Doesn't matter when it's a short list like this but some of the other tests have pretty long lists of options.